car/models.py

```diff
@@ -0,0 +1,54 @@
+from django.db import models
+
+class CarModel(models.Model): 
+    name = models.CharField(max_length = 64)
+
+    class Meta: 
+        db_table = 'car_models'
+
+class Drive(models.Model): 
+    name = models.CharField(max_length = 64)
+
+    class Meta: 
+        db_table = 'drives'
+
+class Battery(models.Model): 
+    name = models.CharField(max_length = 64)
+
+    class Meta: 
+        db_table = 'batteries'
+
+class CarSpecification(models.Model): 
+    model            = models.ForeignKey(CarModel, on_delete = models.CASCADE)
+    drive            = models.ForeignKey(Drive, on_delete = models.CASCADE)
+    battery          = modles.ForeignKey(Battery, on_delete = models.CASCADE)
+    traction_ability = models.IntegerField(null = True)
+    height           = models.IntegerField(null = True)
+    vault_length     = models.IntegerField(null = True)
+    approach_angle   = modles.IntegerField(null = True)
+    deviation_angle  = models.IntegerField(null = True)
+    distance         = models.IntegerField(null = True)
+    load_space       = models.IntegerField(null = True)
+    acceleration     = models.DecimalField(max_digits = 8, decimal_places = 2)
+    top_speed        = models.DecimalField(max_digits = 8, decimal_places = 2)
+    weight           = models.IntegerField(null = True)
+    display_inch     = models.IntegerField(null = True)
+    seats            = models.IntegerField(null = True)
+    warranty         = models.CharField(max_length = 256)
+
+    class Meta:
+        db_table = 'car_specifications'
+
+
+class WheelInch(models.Model):
```

Comment:
추후에, Wheel에 옵션이 추가될 수 있으므로, class명은 Wheel로 변경해주세요.

```suggestion
class Wheel(models.Model):
```

Reply:
네 !! 수정하겠습니다